internal/installercache/installercache.go

```diff
@@ -70,11 +70,7 @@ func (i *Installers) Get(releaseID, releaseIDMirror, pullSecret string, ocReleas
 	var workdir, binary, path string
 	var err error
 
-	releaseImageLocation := releaseID
-	if releaseIDMirror != "" {
-		releaseImageLocation = releaseIDMirror
-	}
-	workdir, binary, path, err = ocRelease.GetReleaseBinaryPath(releaseImageLocation, i.cacheDir, ocpVersion)
+	workdir, binary, path, err = ocRelease.GetReleaseBinaryPath(releaseID, i.cacheDir, ocpVersion)
```

Comment:
Sorry, I'm not sure I understand this change. 

The path here might not match the path generated in the `Extract` function since the `Extract` function could use the mirror image instead, if it's provided. https://github.com/openshift/assisted-service/pull/6918/files#diff-3b8a020ef087b81b6b45d56bef6dd8d32fcf9d50dfe3d91fd369755b253cfff8R372

How come we're only using the release image instead of the mirror image for the path here?

Reply:
The mirror image should be in the ICSP (or IDMS) that is provided (if that is not provided then the explicit mirror image will be used https://github.com/openshift/assisted-service/pull/6918/files#diff-3b8a020ef087b81b6b45d56bef6dd8d32fcf9d50dfe3d91fd369755b253cfff8R484  but the intent is to have the mirror image in the ICSP).

The way its currently implemented is that if the ICSP has multiple entries, all of the entries will not be tried if only one value of mirror image is used.

Reply:
Sure, but sorry, shouldn't this path calculated here match the one in here? https://github.com/openshift/assisted-service/pull/6918/files#diff-3b8a020ef087b81b6b45d56bef6dd8d32fcf9d50dfe3d91fd369755b253cfff8R372

Reply:
OK, I understand the question now. Yes, the call with GetReleaseBinaryPath() will always pass in the releaseImage, not the mirror. Since this path is only used for the cache location its not necessary that it include the mirror string, only that it is the same whenever it is referenced.

Reply:
Ah thanks! I wasn't looking carefully and thought that they were mismatched. Sorry about that!